origin-dapp-creator-server/src/app.js

```diff
@@ -0,0 +1,114 @@
+const bodyParser = require('body-parser')
+const cors = require('cors')
+const express = require('express')
+const Logger = require('logplease')
+const Web3 = require('web3')
+const web3 = new Web3()
+
+const app = express()
+const port = process.env.PORT || 4321
+const logger = Logger.create('origin-dapp-creator-server')
+
+import { getDnsRecord, parseDnsTxtRecord, setAllRecords, updateTxtRecord } from './lib/dns'
+import { addConfigToIpfs, ipfsClient, getConfigFromIpfs } from './lib/ipfs'
+
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true }))
+app.use(cors())
+
+/* Route for handling creation of configurations. Saves the configuration to
+ * IPFS and configures a subdomain if necessary. Subdomains are protected via
+ * web3 signatures so it isn't possible to overwrite another users subdomain.
+ */
+app.post('/config', async (req, res) => {
```

Comment:
Thinking of security. This route is open. Should we add rate limiting as a naive way to mitigate risks of abuse ? Not blocking for getting this PR in.